dataplane/linux/endpoint_mgr.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016-2017 Tigera, Inc. All rights reserved.
+// Copyright (c) 2016-2018 Tigera, Inc. All rights reserved.
```

Comment:
It should only be on a change to a file.  I'll remove this.  (I revved it because it was shown as modified in `git status` but I was doing a merge at the time and it looks like the merge ended up being a no-op.)